server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -60,7 +60,7 @@
   /**
    * Port which the JDBC server listens to.
    */
-  private int rpcPort = 6667;
+  private int rpcPort = 6670;
```

Comment:
Please reverse this.

Reply:
Thanks for your reminding. This is reversed.